Dashboard/Client/SessionManagement/ClientSessionManager.cs

```diff
@@ -217,16 +222,42 @@ public void OnDataReceived(string serializedData)
         /// </summary>
         public void RemoveClient()
         {
-            ClientToServerData clientToServerData = new("removeClient", _user.username, _user.userID);
-            string serializedData = _serializer.Serialize<ClientToServerData>(clientToServerData);
-            _communicator.Send(serializedData, moduleIdentifier);
+            SendDataToServer("removeClient", _user.username, _user.userID);
+        }
+
+        /// <summary>
+        /// A helper function that sends data from Client to the server side. The data consists
+        /// of the event type and the user who requested for that event.
+        /// </summary>
+        /// <param name="eventName"> The type of the event. </param>
+        /// <param name="username"> The username of the user who requested</param>
+        /// <param name="userID"> The user ID of the user, if the user has not yet beend created then the default value is -1. </param>
+        public void SendDataToServer(string eventName, string username, int userID = -1)

```

Comment:
Why is this function public... Is it used in outside scope?

Reply:
No, I will make the required changes.